crates/compiler/solve/src/ability.rs

```diff
@@ -463,6 +463,13 @@ fn is_builtin_int_alias(symbol: Symbol) -> bool {
         | Symbol::NUM_I32  | Symbol::NUM_SIGNED32
         | Symbol::NUM_I64  | Symbol::NUM_SIGNED64
         | Symbol::NUM_I128 | Symbol::NUM_SIGNED128
+    )
+}
+
+#[inline(always)]
+#[rustfmt::skip]
+fn is_builtin_nat_alias(symbol: Symbol) -> bool {
+    matches!(symbol,
         | Symbol::NUM_NAT  | Symbol::NUM_NATURAL
     )
 }
```

Comment:
what is the problem with formatting here? how is that leading `|` in a pattern valid?

Reply:
good call this should be taken out. I more so want to skip the formatting for `is_bultin_fixed_int` because otherwise rustfmt makes it hard to read

Reply:
Fixed in [63e3044](https://github.com/roc-lang/roc/pull/4974/commits/63e30443fde725373608bee7589df439b0047df2)